todo_app/app.py

```diff
@@ -1,11 +1,28 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items 
+from todo_app.data.session_items import add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', my_items = items)
+
+@app.route('/add_new_items', methods=['POST'])
+def add_new_item():
+    new_item_title = request.form.get('Title')
+    add_item (new_item_title)
+    return redirect('/')
+
+
+
+
+#@app.route('/add_new_items', methods=['POST'])
+#def additem():
+#    add_item = request.form.get('New Item Title')
+#    return redirect ???????
```

Comment:
Just a reminder to delete this unused code and extra whitespace at a later stage